sources/packages/web/src/views/institution/EditInstitutionLocation.vue

```diff
@@ -8,6 +8,17 @@
         data-cy="editLocationHeader"
       />
     </template>
+    <template #alerts>
+      <banner
+        :type="BannerTypes.Info"
+        header="How to update information"
+        summary="Please email a request to update the location details. For the primary contact, you can update it without a request."
+      >
+        <template #actions>
+          <v-btn color="info">Email designat@gov.bc.ca</v-btn>
```

Comment:
Can you please clarify what is the idea behind this button?
I believe that we are now using it at least in two places para it is actually not doing anything.

Reply:
Thanks for the clarifications, Lynn.
Maybe we can have a clickable link to open the email client and a clipboard icon to copy it to the clipboard.